src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -655,7 +657,7 @@ Type[] templateArguments(Context context) throws ParserException {
                 break;
             }
         }
-        return arguments.toArray(new Type[arguments.size()]);
+        return arguments.toArray(new Type[0]);
```

Comment:
All these kind of changes follow this recommendation: "In older Java versions, using a pre-sized array was recommended, as the reflection call necessary to create an array of proper size was quite slow.
However, since late updates of OpenJDK 6, this call was intrinsified, making the performance of the empty array version the same, and sometimes even better, compared to the pre-sized version. Also, passing a pre-sized array is dangerous for a concurrent or synchronized collection as a data race is possible between the size and toArray calls. This may result in extra nulls at the end of the array if the collection was concurrently shrunk during the operation."